test/cql-pytest/test_tools.py

```diff
@@ -834,3 +834,34 @@ def test_scrub_validate_mode(scylla_path, scrub_workdir, scrub_schema_file, scru
 
         # Check that validate did not move the bad sstable into qurantine
         assert os.path.exists(scrub_bad_sstable)
+
+def flush_with_nodetool_utils(cql, ks=None, table=None):
+    if not ks:
+        nodetool.flush(cql)
+    elif not table:
+        nodetool.flush(cql, ks)
+    else:
+        nodetool.flush(cql, f"{ks}.{table}")
+
+def flush_with_scylla_nodetool(cql, scylla_path, ks=None, table=None):
+    cmd = [scylla_path, "nodetool", "flush"]
+    if ks:
+        cmd.append(ks)
+        if table:
+            cmd.append(table)
+    cmd.extend(["--host", cql.cluster.contact_points[0]])
+    subprocess.check_call(cmd)
+
+def test_flush_table(cql, test_keyspace, scylla_path, scylla_data_dir):
+    with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir, lambda cql, ks, tbl: flush_with_nodetool_utils(cql, ks, tbl)) as (schema_file, sstables):
+        assert sstables != []
+
+    with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir, lambda cql, ks, tbl: flush_with_scylla_nodetool(cql, scylla_path, ks, tbl)) as (schema_file, sstables):
+        assert sstables != []
+
+def test_flush_keyspace(cql, test_keyspace, scylla_path, scylla_data_dir):
+    with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir, lambda cql, ks, _: flush_with_nodetool_utils(cql, ks)) as (schema_file, sstables):
+        assert sstables != []
+
+    with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir, lambda cql, ks, _: flush_with_scylla_nodetool(cql, scylla_path, ks)) as (schema_file, sstables):
+        assert sstables != []
```

Comment:
These tests don't really belong here. We have `tests/nodetool` for front-end tests and nodetool tests in dtest.
This test-file is for testing scylla-sstable.

Reply:
ok, it's not quite clear that `test_tools.py` is testing only scylla-sstable.
maybe it'd be better to rename it accordingly.

Reply:
Yes, renaming is probably a good idea, to avoid confusion.

Reply:
let's rename in a followup PR